CHANGES.txt

```diff
@@ -5,6 +5,41 @@ We only cover here the main highlights, for a detailed list of all the changes
 see `the github commits changelog
 <https://github.com/hyperspy/hyperspy/commits/master>`_.
 
+v0.8.5
+======
+
+
+This is a maintenance release. Follow the following links for details on all
+the `bugs fixed
+<https://github.com/hyperspy/hyperspy/issues?q=is%3Aissue+milestone%3Anext_maintenance_release+label%3Abug+is%3Aclosed>`_,
+`feature <https://github.com/hyperspy/hyperspy/issues?utf8=%E2%9C%93&q=milestone%3Anext_maintenance_release+is%3Aclosed++label%3Aenhancement+>`_
+and `documentation
+<https://github.com/hyperspy/hyperspy/pulls?utf8=%E2%9C%93&q=milestone%3Anext_maintenance_release+label%3Adocumentation+is%3Aclosed+>`_ enhancements.
+
+
+It also includes one new features and introduces an important API change that
```

Comment:
grammar --- It also includes a new feature and introducees an important API change